core/tooling/src/main/resources/akka-microservices-crd.yml

```diff
@@ -0,0 +1,466 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
Why is this crd here?

Reply:
Because the GraalVM automatic configuration should reach as many code-paths as possible, I forgot to add this step to the training while integrating the Microservices support

Reply:
ok